lib/voxpupuli/test/rake/spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'puppet_fixtures/tasks'
+require 'parallel_tests'
+require 'rspec/core/rake_task'
+
+pattern = 'spec/{aliases,classes,defines,functions,hosts,integration,plans,tasks,type_aliases,types,unit}/**/*_spec.rb'
+
+desc 'Run spec tests and clean the fixtures directory if successful'
+task spec: :'fixtures:prep' do |_t, args|
+  Rake::Task['spec:standalone'].invoke(*args.extras)
+  Rake::Task['fixtures:clean'].invoke
+end
+
+desc 'Run spec tests in parallel and clean the fixtures directory if successful'
+task :parallel_spec do |_t, args|
+  Rake::Task['fixtures:prep'].invoke
```

Comment:
YOu can do the same trick as with the regular spec and make it a dependency
```suggestion
task parallel_spec: :'fixtures:prep' do |_t, args|
```